arangod/RestHandler/RestGraphHandler.cpp

```diff
@@ -928,7 +940,7 @@ Result RestGraphHandler::graphActionCreateGraph() {
   bool parseSuccess = false;
   VPackSlice body = this->parseVPackBody(parseSuccess);
   if (!parseSuccess) {
-    return {TRI_ERROR_BAD_PARAMETER, "unable to parse body"};
+      return returnError(TRI_ERROR_BAD_PARAMETER, "unable to parse body");
```

Comment:
```suggestion
    return returnError(TRI_ERROR_BAD_PARAMETER, "unable to parse body");
```
no guarantee that this is correct, looked like it in the UI